src/MissionCreator.sol

```diff
@@ -0,0 +1,199 @@
+// SPDX-License-Identifier: MIT
```

Comment:
looks like there's a bunch of temporary/testing values in here. might be useful to add //TODO or //FIXME on those parts.